lib/power_action_utils.pm

```diff
@@ -127,8 +127,16 @@ sub poweroff_x11 {
 
     if (check_var("DESKTOP", "lxqt")) {
         # opens logout dialog
-        x11_start_program('shutdown', target_match => 'lxqt_logoutdialog');
-        send_key "ret";
+        x11_start_program('shutdown', target_match => [qw(authorization_failed lxqt_shutdowndialog)]);
+        # we have typing issue because of poor performance, to record this if happens.
+        # close authorization failed dialog caused by bsc#1137230
```

Comment:
According to https://openqa.opensuse.org/tests/837926#step/shutdown/2, it was an `authorization_required` check, so, why it is an `authorization failed`, now?

Reply:
I think this is because CD-ROM pop-up got typing and authorization failed displayed now. so this is related to pop-up